tests/publiccloud/run_ltp.pm

```diff
@@ -47,21 +45,20 @@ sub instance_log_args
 
 sub upload_ltp_logs
 {
-    my ($self) = @_;
     record_info('LTP Logs', 'upload');
-    assert_script_run("test -f $root_dir/result.json || echo No result log");
-    parse_extra_log('LTP', "$root_dir/result.json");
-    # debug file in the standart LTP log-dir. structure:
-    assert_script_run("test -f /tmp/runltp.\$USER/latest/debug.log || echo No debug log");
-    upload_logs("/tmp/runltp.\$USER/latest/debug.log", failok => 1);
+    assert_script_run("test -f /tmp/kirk.\$USER/latest/debug.log || echo No debug log");
+    upload_logs("/tmp/kirk.\$USER/latest/debug.log", log_name => 'debug.log.txt', failok => 1);
 
-    die $@ if $@;
+    assert_script_run("test -f /tmp/kirk.\$USER/latest/results.json || echo No results file");
```

Comment:
Using `assert` with `OR` inside,it makes no sense to me because it seems never fails: I agree with Anton :)